shiny/ui/_theme.py

```diff
@@ -516,21 +555,26 @@ def path_pkg_preset(preset: ShinyThemePreset, *args: str) -> str:
     return pathlib.Path(path).as_posix()
 
 
-def check_is_valid_preset(preset: ShinyThemePreset) -> None:
+def check_is_valid_preset(preset: str | None) -> ShinyThemePreset:
     if preset not in shiny_theme_presets:
         raise ValueError(
             f"Invalid preset '{preset}'.\n"
             + f"""Expected one of: "{'", "'.join(shiny_theme_presets)}".""",
         )
 
+    return preset
 
-def check_libsass_installed() -> None:
+
+def check_theme_pkg_installed(pkg: str, spec: str | None = None) -> None:
     import importlib.util
 
-    if importlib.util.find_spec("sass") is None:
+    if spec is None:
+        spec = pkg
+
+    if importlib.util.find_spec(spec) is None:
         raise ImportError(
-            "The 'libsass' package is required to compile custom themes. "
-            'Please install it with `pip install libsass` or `pip install "shiny[theme]"`.',
+            f"The '{pkg}' package is required to compile custom themes. "
+            'Please install it with `pip install {pkg}` or `pip install "shiny[theme]"`.',
```

Comment:
This is gonna suggest `pip install brand_yml` if not installed already

Reply:
It might be missing something but that's the intention, yeah. Since the theming packages are soft dependencies we need to prompt users to install `brand_yml` or `libsass` if not included (and one or the other might be on its own).

Reply:
Oh, I hadn't realized that `pip install brand_yml` actually works -- I thought it had to be `pip install brand-yml`